### DISCUSSION TITLE
Install Emgu 4.5.4.4788 for .NET Framework?

### DISCUSSION THREAD

**Discussion by RecumbentGourmet** [emgucv/emgucv#677] [Q&A]
HI - I'm new to Emgu, and I'm quite frustrated attempting to get Emgu installed for a Winforms project that targets the .NET framework (V4.7.2).  When I attempt to use NuGet to install Emgu into my project, it fails with an error saying that it required .NET core (V5). 

 'Emgu.runtime.windows.msvc.rt.x64 19.28.29336'. You are trying to install this package into a project that targets '.NETFramework,Version=v4.7.2', but the package does not contain any assembly references or content files that are compatible with that framework

 I've read on the web about issue with 'Package.Config', but you can see in my solution in Visual Studio 2019 I do not have any packages listed so there is not an option to migrate my solution to the different package reference configuration.
 
 So - it is possible to build a new Winform project using .NET framework that references the latest build on Emgu?   If so, how? 
 
 
![image](https://user-images.githubusercontent.com/17932558/146987407-7e1ee387-8a54-4a4b-83de-e5349417e5f9.png)

**Comment by emgucv** [comment]
Looks like you are creating a VB project with .NetFramework 4.7.2. 

I tried and it seems that VB project do not allow you to switch to use the Package Reference to manage nuget packages. You can only change from "Package.config" to "Package Reference" on .NetFramework 4.7.2 when using a C# project.

If using .NetFramework 4.7.2 is a requirement, you can switch to C# project. 

If you can use the newer .Net Core 3.0 / .Net 5.0+ you can still use VB.

If you absolutely need to use VB project and .Net Framework 4.7.2, you cannot use nuget. You will need to download the ".zip" release package and add reference to a shared project:
https://www.emgu.com/wiki/index.php/Download_And_Installation#Open_CV_unmanaged_dll
